backend/infrahub/core/migrations/graph/m019_restore_rels_to_time.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Sequence
+
+from infrahub.core.migrations.shared import MigrationResult
+from infrahub.log import get_logger
+
+from ..shared import InternalSchemaMigration, SchemaMigration
+
+if TYPE_CHECKING:
+    from infrahub.database import InfrahubDatabase
+
+log = get_logger()
+
+
+class Migration019(InternalSchemaMigration):
+    name: str = "019_restore_relationships_to_time"
+    minimum_version: int = 18
+    migrations: Sequence[SchemaMigration] = []
+
+    async def validate_migration(self, db: InfrahubDatabase) -> MigrationResult:
+        result = MigrationResult()
+
+        return result
+
+    async def execute(self, db: InfrahubDatabase) -> MigrationResult:
+        """
+        Fix corrupted state introduced by Migration012 when duplicating a CoreAccount (branch Aware)
+        being part of a CoreStandardGroup (branch Agnostic). Database is corrupted at multiple points:
+        - Old CoreAccount node <> group_member node `active` edge has no `to` time (possibly because of #5590).
```

Comment:
Actually, #5590 deserves a dedicated migration fixing missing `to` time for any couple of `deleted` edge on `-global-` with an `active` edge on another branch.